### PR TITLE
Bug 1881573: remove use of imminent SCL EOL images

### PR DESCRIPTION
**Pull request by gabemontero**
See https://github.com/openshift/cluster-samples-operator/pull/326#issuecomment-697045304

### CODE REVIEW

File: test/extended/testdata/builds/build-pruning/default-group-build-config.yaml [hunk 1]
```diff
@@ -13,4 +13,4 @@ spec:
       from:
         kind: ImageStreamTag
         namespace: openshift
-        name: 'php:7.2'
+        name: '7.3-ubi8'
```
Comment (yselkowitz) [inline test/extended/testdata/builds/build-pruning/default-group-build-config.yaml:16]: php:7.3-ubi8 ??

File: test/extended/testdata/builds/build-pruning/default-legacy-build-config.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@
         "from": {
           "kind": "ImageStreamTag",
           "namespace": "openshift",
-          "name": "php:7.2"
+          "name": "7.3-ubi8"
```
Comment (yselkowitz) [inline test/extended/testdata/builds/build-pruning/default-legacy-build-config.yaml:20]: php:7.3-ubi8 ??

File: test/extended/testdata/builds/build-pruning/errored-build-config.yaml [hunk 3]
```diff
@@ -20,4 +20,4 @@ spec:
       from:
         kind: ImageStreamTag
         namespace: openshift
-        name: 'php:7.2'
+        name: '7.3-ubi8'
```
Comment (yselkowitz) [inline test/extended/testdata/builds/build-pruning/errored-build-config.yaml:23]: php:7.3-ubi8 ??

File: test/extended/testdata/builds/build-pruning/failed-build-config.yaml [hunk 4]
```diff
@@ -18,4 +18,4 @@ spec:
       from:
         kind: ImageStreamTag
         namespace: openshift
-        name: 'php:7.2'
+        name: '7.3-ubi8'
```
Comment (yselkowitz) [inline test/extended/testdata/builds/build-pruning/failed-build-config.yaml:21]: php:7.3-ubi8 ??

**Review by yselkowitz — CHANGES_REQUESTED** [review]

File: test/extended/testdata/builds/build-pruning/default-group-build-config.yaml (hunk 1 above)
Comment (gabemontero) [reply test/extended/testdata/builds/build-pruning/default-group-build-config.yaml:16]: yep

**Review by adambkaplan — APPROVED** [review]
/lgtm

**Pull request merged**